src/assets/ts/definitions/basics.ts

```diff
@@ -255,6 +255,38 @@ keyDefinitions.registerAction(new Action(
   },
 ));
 
+keyDefinitions.registerAction(new Action(
+  'visual-line-join',
+  'Join all selected rows',
+  async function({ session, visual_line }) {
+    if (visual_line == null) {
+      throw new Error('Visual_line mode arguments missing');
+    }
+
+    if (visual_line.num_rows >= 1) {
```

Comment:
should it work even if there's only one row?  then will just join all the children

Reply:
Yes, that was the goal - to join child nodes.

Reply:
sorry, I misread (thought it was >1).  Does == 0 happen as well?  this looks good